### DISCUSSION TITLE
无互联网连接或清单不可用

### DISCUSSION THREAD

**Discussion by qingzt** [pjy612/SteamManifestCache#23] [Question / Answer]
大部分情况是无互联网连接，有时候是清单不可用
已经确定过了，清单是最新的
![image](https://github.com/pjy612/SteamManifestCache/assets/116171689/d11aa4fe-41b2-4418-aa13-5ce6fd76b092)
![image](https://github.com/pjy612/SteamManifestCache/assets/116171689/4fbc590f-5c89-47b0-841c-02f42b596e9c)
![image](https://github.com/pjy612/SteamManifestCache/assets/116171689/f8f9b022-53dd-4256-913a-73e0d7673c59)
![image](https://github.com/pjy612/SteamManifestCache/assets/116171689/5e811dfb-8121-419f-b219-d0e7b670022e)
也有密钥
![image](https://github.com/pjy612/SteamManifestCache/assets/116171689/d7a6030a-4aea-4c1b-ad18-905e40a4bbdc)
求大佬解答

**Comment by qingzt** [comment]
似乎搞懂了，卸载之后重新安装没有问题，难道更新游戏需要旧的清单与新的清单并存？

**Reply by pjy612** [reply]
清单不可用就是 清单旧了。
网络不可用 和 无连接 参考 https://github.com/pjy612/SteamManifestCache/discussions/11
内容加密 就是 密钥写入失败，推荐关steam 操作。
